consult.el

```diff
@@ -236,10 +236,12 @@ See `consult--multi' for a description of the source data structure."
   "Regexp used to match file and line of grep output.")
 
 (defcustom consult-grep-args
-  "grep --null --line-buffered --color=never --ignore-case --line-number -I -r ."
+  '("grep" (consult--grep-generate-exclude-args-common)
+    "--null --line-buffered --color=never\
+    --ignore-case --line-number -I -r .")
   "Command line arguments for grep, see `consult-grep'.
```

Comment:
The docstring should be expanded. It should mention both possibilities - strings and lists with evaluated arguments.

Reply:
done